features/step_definitions/logging.rb

```diff
@@ -1056,3 +1056,125 @@
     | pod_label       | #{pod_label}     |
   })
 end
+
+
+
+Given /^I have(?: "(\w+)")? log pod in project #{QUOTED}$/ do | log_type, project_name |
+    log_type ||= "any"
+    file_dir = "#{BushSlicer::HOME}/testdata/logging/loggen"
+
+    unless project(project_name).exists?
+        step %Q/I run the :new_project client command with:/,table(%{
+          | project_name | #{project_name} |
+        })
+        raise "Error creating namespace" unless @result[:success]
+    end
+
+    case log_type
+      when "json"
+        template_file = "#{file_dir}/container_json_log_template.json"
```

Comment:
From the whole `case` block, I think no need to declare `json` here.

Reply:
I'd like keep the 'json' declaration.  That make the function more specific.